usr/share/rear/prep/DP/default/450_check_dp_client_configured.sh

```diff
@@ -1,15 +1,19 @@
 # 450_check_dp_client_configured.sh
 # this script has a simple goal: check if this client system has been properly
-# defined on the DP cell server and that a backup specification has been
-# made for this client - no more no less
+# configured on the Cell Manager and backed up at least once - no more no less
 
 OMNIDB=/opt/omni/bin/omnidb
 OMNIR=/opt/omni/bin/omnir
+VBDA=/opt/omni/lbin/vbda
 
 Log "Backup method is DP: check Data Protector requirements"
+[ -x ${VBDA} ]
+StopIfError "Please install Data Protector Disk Agent (DA component) on the client."
+
 [ -x ${OMNIR} ]
-StopIfError "Please install Data Protector User Interface (cc component) on the client."
+StopIfError "Please install Data Protector User Interface (CC component) on the client."
 
-${OMNIDB} -filesystem | grep $(hostname) >/dev/null
-StopIfError "Data Protector check failed with error code $? (no filesystem backup found).
+${OMNIDB} -filesystem | grep $(hostname)
+StopIfError "Data Protector check failed, error code $?.
+Check if the user root is configured in Data Protector UserList and if backups for this client exist in the IDB.
```

Comment:
Same as https://github.com/rear/rear/pull/2443/files#r448166732

Using `$?` in the `StopIfError` function message does not work as intended.